.github/workflows/cd-publish-packages.yml

```diff
@@ -0,0 +1,56 @@
+name: CD - FAST Tooling Publisher
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 7 * * 0-4'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GITHUB_SERVICE_USER: "Microsoft FAST DevOps"
+  GITHUB_SERVICE_EMAIL: "fastsvc@microsoft.com"
+      
+jobs:
+  build_linux:
+    runs-on: ubuntu-latest
+    if: contains('janechu,williamw2,awentzel,eisenbergeffect', github.actor)
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        token: ${{ env.GITHUB_TOKEN }}
+
+    - name: Add or Update packages
+      run: sudo yarn global add lerna
+
```

Comment:
Neither Lerna or Yarn are used in this package, I would also not update the packages

```suggestion
```